tldw_Server_API/app/core/DB_Management/Jobs_Repository.py

```
@@ -0,0 +1,240 @@
+either manage their own transaction scope or reuse a caller-supplied session.
+"""
+
+from __future__ import annotations
+
+import contextlib
+import sqlite3
+import uuid as _uuid
+from dataclasses import dataclass
+from datetime import datetime, timezone as _tz
+from pathlib import Path
+from typing import Any, Iterator
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)